### ISSUE TITLE
Example of `geometry-script` in a standalone python script?

### ISSUE THREAD

**Issue by abegong** [carson-katri/geometry-script#9]
Great project! I'm excited to see where you go with this.

I can imagine a lot of powerful use cases for `geometry-script` from the command line, but all of the examples that I've been able to find assume that it'll be used from within the Blender application.

How would I run `geometry-script` in a standalone python script?

**Comment by carson-katri** [comment]
Geometry Script creates Blender node trees, so it doesn't make sense to run it outside of Blender imo. Unless you are thinking about generating the `.blend` file data for the Geometry Node tree via the command line?

**Comment by abegong** [comment]
Yes, that'd be a good way to do it. That would let you mash up the .blend files with programmatically generated scenes and objects.
